### DISCUSSION TITLE
What is the recommended way to query geolocation ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [microsoft/microsoft-ui-xaml#9453] [Q&A]
[MAUI](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/device/geolocation?view=net-maui-8.0&tabs=windows) has API and so does UWP. Is there a way to achieve this and avoid specifying Windows UWP-like contracts ?

**Comment by DarranRowe** [comment]
Why do you want to avoid UWP like contracts? Especially when you are asking a question in the Microsoft.UI.Xaml repository. WinUI 2 requires running in a UWP Xaml application and WinUI 3 requires the usage of UWP like contracts.

**Reply by sigmarsson** [reply]
Sorry I might have phrased inappropriate;  I intend to utilize the `Windows.Devices.Geolocation.Geolocator` class in a library without specifying any (specific) moniker; `<TargetFrameworks>net8.0</TargetFrameworks>` because some other cmd line tool also referencing this lib. 
Earlier I invoekd it from `Uno.WinUI.dll` but it is no longer the case.  IF this is not feasible, what nuget shall I add ?
